libfdisk/src/dos.c

```diff
@@ -2597,6 +2597,73 @@ static int dos_reorder(struct fdisk_context *cxt)
 	return 0;
 }
 
+int fdisk_dos_fix_chs_values(struct fdisk_context *cxt)
```

Comment:
What about "fdisk_dos_fix_chs()" name?

You also need /** ... */ comment before the function to make it visible for  libfdisk/docs/.

Reply:
Question is whether it should be exported libfdisk function or if it should be callback function inside the `struct fdisk_label_operations`.

Reply:
I'd like to avoid it as a callback, it's really dos specific. The operations are for function which we can assume for more labels.

Reply:
Ok!